### DISCUSSION TITLE
Xem xét lại Dimensionality Reduction

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#80] [Terminology] [answered]
Từ `dimensionality` mình có bàn ở d2l từ trước (https://github.com/mlbvn/d2l-vn/pull/1582), và đang để là `kích thước chiều`.
Tuy nhiên cụm `dimensionality reduction` được dịch rộng rãi là `giảm chiều dữ liệu` chứ không phải `giảm kích thước chiều`, mọi người thấy thế nào nhỉ?

cc @mlbvn/handson-ml2

**Comment by cuongvng** [comment]
nếu vậy thêm "Dimensionality Reduction" là "giảm chiều dữ liệu", còn vẫn giữ nguyên "dimensionality" cho các context khác thì sao?

**Reply by ngcthuong** [reply]
Dimensionality reduction là giảm chiều/giảm số chiều chứ không có đi kèm dữ liệu hay gì cả. Mình có thể dịch linh động theo từng trường hợp tương ứng. Như trong signal processing, ko có dữ liệu mà chỉ có tín hiệu thôi.

**Comment by minhduc0711** [comment]
Nhân tiện thì mình có thể sửa lại `dimensionality` thành `chiều` luôn cho ngắn gọn, vì giờ mình đọc lại conversation cũ cũng thấy không hợp lý cho lắm (pooling có làm giảm dimensionality của 1 điểm dữ liệu).

**Reply by rootonchair** [reply]
Em không ủng hộ cách dịch này cho lắm. `chiều` trong tiếng Việt là một từ đa nghĩa "dimension", "direction" nên đôi thi sẽ cần thêm `số chiều` để tránh nhập nhằng

**Reply by minhduc0711** [reply]
Mình nghĩ có context sẽ không sao, `giảm số chiều dữ liệu` nghe hơi dài dòng.

**Reply by cuongvng** [reply]
Chắc vẫn linh động tùy context được, cần thì viết đủ

**Comment by minhduc0711** [comment, marked as answer]
https://github.com/mlbvn/handson-ml2-vn/discussions/80#discussioncomment-627132
> Dimensionality reduction là giảm chiều/giảm số chiều chứ không có đi kèm dữ liệu hay gì cả. Mình có thể dịch linh động theo từng trường hợp tương ứng. Như trong signal processing, ko có dữ liệu mà chỉ có tín hiệu thôi.
